src/main/java/frc/robot/commands/RelinquishCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Acquisition;
+
+public class RelinquishCommand extends Command {
+    private Acquisition acquisition = Acquisition.getInstance();
+    private Timer timer = new Timer();
+    private double timeToDrop = 1;
+
+    public RelinquishCommand() {
+        addRequirements(acquisition);
```

Comment:
You can create an instance called "time to drop" without initializing it and then set its value to the default constant in the constructor.

Reply:
we're adding the requirement for the subsystem we're using, what do you mean by this?

Reply:
Rather than initializing the instance within the instance field, I would create a constant in the corresponding constant file storing the default value. Then, within the constructor, set the time to drop variable to this default value.